### DISCUSSION TITLE
Security -> Safety / Protection mode?

### DISCUSSION THREAD

**Discussion by HACS-bank** [jmcollin78/versatile_thermostat#274] [Ideas]
I had a lot of trouble understanding "security" mode until I learned that the original language for this excellent tool is French. 

In English, "security" has a slightly different meaning than "sécurité"; a "security alarm" is a burglar alarm. 

A better translation is probably "safety" or "protection" mode.

Best wishes

**Comment by jmcollin78** [comment]
You are so right.. I will change the documentation and labels.

**Comment by jmcollin78** [comment]
Done (in next release). But it will be misleading : attribute are named `security_xxxx` (I cannot change this -> backward compatibility will be broken) but the text speaks about safety.

I hope it will be clear for English people ...

**Reply by teeten** [reply]
Hello JM,

Compatibility is important, so you could create a second attribute **safety_xxxx** for each value of **xxxx** and keep the **security_xxxx** attribute in sync, and/or make your software check for **safety_xxxx** first, and fall back to **security_xxxx** if it does not exist. This is probably easier said than done, and I have not developed for HA yet so I can't tell if it would be overkill or not. Just my 2 cents.

Anyway, thanks for the great work, this is the best and only HA thermostat I am using for my apartment (electric heating only, just using Zigbee relays and temperature sensors). Love the idea of controlling the duty cycle, and staging when controlling multiple heaters so they do not all start at the same time. Very smart.

JC

**Reply by jmcollin78** [reply]
Yes of course, if I had time and money. But my main problem is more how to find time to do the backlog and fix the bugs. Renaming an internal variable have no business value and a great risk of regression.

Maybe for the next major release which can breaks ascendant compatibility this could be the right time to do that.
